test/kethernft.js

```diff
@@ -133,7 +135,7 @@ describe('KetherNFT', function() {
     await KNFT.connect(account1).wrap(idx, await account1.getAddress());
     {
       const expected = {
-        "name": "ThousandEtherHomepage #0: 30x40 at [10,20]",
+        "name": "Ad #0: 30x40 at [10,20]",
```

Comment:
@mveytsman FYI changing the name in the metadata to be shorter so it doesn't get truncated in OpenSea UI.

Reply:
Oh looks like OpenSea made it wrap now, but still it's redundant with the same string just above it

![image](https://user-images.githubusercontent.com/6292/139502772-7f34b61d-6f78-4c70-877b-f06f1f54c4b6.png)